Update USD to 20.08

jcowles opened the pull request.

jcowles committed:
USD 20.08: Update BUILDING.md

jcowles committed:
USD 20.08: Update minimal build command

jcowles committed:
USD 20.08: Update Python build scripts for Python3

jcowles committed:
USD 20.08: Update Swig and third_party headers

jcowles committed:
USD 20.08: Add property API exports for UsdCs.dll

jcowles committed:
USD 20.08: Upgrade UsdCs to VS2019, Win10

jcowles committed:
USD 20.08: Fix assembly info file

jcowles committed:
USD 20.08:  Regenerate bindings & libs

jcowles committed:
USD 20.08: Fix MaterialSample required by API changes

jcowles committed:
USD 20.08: Update USD libs for Windows

jcowles committed:
USD 20.08: Update libs for final USD 20.08 build

jcowles commented:
The Yamato error is also happening with older binaries, so I have to assume it's something funky with trunk (20.2).

I'm looking into it.

judubu commented:
Thanks for the big update. 
Do you have the resources to build for OSX and Linux? I wanted to get the build infra back up and running mid when I'm back in August.
The trunk jobs on Yamato started failing beginning of June with all errors stating USD.NET not being loadable. Looked into it a little bit but came back empty handed. Completely forgot about it with the Chaplin rush.

jcowles commented:
I can produce an OSX build (manually via XCode), but I have no local ability to create a Linux build.

I started setting up locally to start to learn our new build infrastructure (for automated win/linux/osx builds), though I'm now waiting for that team to process me as a new user.

jcowles committed:
Serialization: Use CLS compliant IntPtr conversion

jcowles committed:
USD 20.08: Update DLLs after CLS change

judubu committed:
USD v20.08: Build UsdCs for Linux

jcowles committed:
USD 20.08: Update initUsd for new plug info path

jcowles committed:
USD 20.08: Disable xcode code signing

jcowles committed:
USD 20.08: Update OSX libs

jcowles committed:
USD 20.08: Update USD version in package manifest

jcowles committed:
USD 20.08: Update package version to 1.0.4

jcowles committed:
USD 20.08: Update Changelog

jcowles committed:
USD 20.08: Clean up share folder in bundle

jcowles committed:
USD 20.08: Update OSX libs with correct versions

Alembic version was incorrect.
Updated dependent libs for good measure.

jcowles committed:
USD 20.08: Remove unused plugInfo files

jcowles committed:
USD 20.08: Remove OSX-specific pluginfo files

These used to be required because .dylib was burned into them, however this is no longer the case.

jcowles committed:
USD 20.08: Update initUsd to share common plugInfo path for OSX
